discord/message.py

```diff
@@ -1937,6 +1969,11 @@ def _handle_components(self, data: List[ComponentPayload]) -> None:
     def _handle_interaction(self, data: MessageInteractionPayload):
         self.interaction = MessageInteraction(state=self._state, guild=self.guild, data=data)
 
+    def _handle_poll_vote(self, answer_id: int, added: bool):
```

Comment:
Is there any reason for this function to exist?
Why can't the state just mess with the poll directly?